cmd/loop/loopout.go

```diff
@@ -73,6 +73,10 @@ var loopOutCommand = cli.Command{
 				"Not setting this flag therefore might " +
 				"result in a lower swap fee.",
 		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Assumes yes during confirmation. Using this " + "option will result in an immediate loop in",
```

Comment:
s/"immediate loop in"/ "immediate loop out"

Could also just say "immediate swap", declare the flag as a var then reuse across commands.